tests/components/test_question_answerer.py

```diff
@@ -63,12 +61,53 @@ def food_ordering_answerer(food_ordering_app_path, es_client):
         index_name="menu_items",
         data_file=DISH_DATA_FILE_PATH,
     )
-    es_client.indices.flush(index="_all")
 
     qa = QuestionAnswerer(food_ordering_app_path)
     return qa
 
 
+@pytest.mark.extras
+@pytest.fixture
+def food_ordering_with_bert(food_ordering_app_path, es_client):
+    bert_qa_config = {
+        "model_type": "embedder",
+        "model_settings": {
+            "embedder_type": "bert",
+            "embedding_fields": {"menu_items_bert": ["name"]},
+        },
+    }
+    QuestionAnswerer.load_kb(
+        app_namespace="food_ordering",
+        index_name="menu_items_bert",
+        data_file=DISH_DATA_FILE_PATH,
+        config=bert_qa_config,
+    )
+
+    qa = QuestionAnswerer(food_ordering_app_path, config=bert_qa_config)
+    return qa
+
+
+@pytest.mark.extras
+@pytest.fixture
+def food_ordering_with_glove(food_ordering_app_path, es_client):
+    glove_qa_config = {
+        "model_type": "embedder",
+        "model_settings": {
+            "embedder_type": "glove",
+            "embedding_fields": {"menu_items_glove": ["name"]},
```

Comment:
@arushir  Do you think it's intuitive for developers to specify the field name derived from the entity type ie `menu_items` and MindMeld automatically mapping that to the correct field name `menu_items_glove `based on the `embedder_type `? Or do you think it's better for the developer to explicitly specify `menu_items_glove `?

Reply:
The key in the `embedding_fields` dictionary is the name that the developer gives their index. If the name of their index is `menu_items`, they would specify `menu_items` here. I just used `menu_items_glove` because I created two different indices for the tests (see pytest fixtures above), one with glove embeddings and another for bert embeddings